homeassistant/components/sentry/sensor.py

```diff
@@ -0,0 +1,68 @@
+"""Support for Sentry SDK sensor."""
+import logging
+
+import sentry_sdk
+from sentry_sdk.integrations.logging import LoggingIntegration
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_NAME
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity import Entity
+
+from .const import CONF_DSN, CONF_ENVIRONMENT
+
+
+class Sentry(Entity):
+    """Representation of the Sentry SDK."""
+
+    def __init__(self, hass: HomeAssistant, config: dict):
+        """Initialize Sentry sensor."""
+        self.hass = hass
+
+        self._name = config.get(CONF_NAME) or "sentry"
+        self._last_event_id = None
+
+        # https://docs.sentry.io/platforms/python/logging/
+        sentry_logging = LoggingIntegration(
+            level=logging.INFO,  # Capture info and above as breadcrumbs
+            event_level=logging.ERROR,  # Send errors as events
+        )
+
+        sentry_sdk.init(
```

Comment:
We should move the Sentry init to the setup entry method in `sentry/__init__.py`. We don't want constructors to have side effects and don't want platforms to do anything else besides provide entities.

Reply:
I will take a stab at removing the entity and just using the single hook.